net/asterisk-16.x/Makefile

```diff
@@ -0,0 +1,1035 @@
+#
+# Copyright (C) 2017 - 2018 Jiri Slachta <jiri@slachta.eu>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+AST_MAJOR_VERSION:=16
+PKG_NAME:=asterisk$(AST_MAJOR_VERSION)
+PKG_VERSION:=$(AST_MAJOR_VERSION).2.1
```

Comment:
There is currently 16.3.0 available. Would you be so kind to update it?

Reply:
Also done. New module res_mwi_devstate was added. Added it to Makefile as well. Run-tested, gave it a quick spin on the router.